### PR TITLE
Update on index reference documentation

### PR DESCRIPTION
**Pull request by yuexiao-wang**
Update on index reference documentation which is inconsistent with command usage

Signed-off-by: yuexiao-wang wang.yuexiao@zte.com.cn

### CODE REVIEW

File: docs/reference/commandline/index.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ read the [`dockerd`](dockerd.md) reference page.
 |:--------|:-------------------------------------------------------------------|
 | [dockerd](dockerd.md) | Launch the Docker daemon                             |
 | [info](info.md) | Display system-wide information                            |
-| [inspect](inspect.md)| Return low-level information on a container or image  |
+| [inspect](inspect.md)| Return low-level information by one or multiple name or ID |
```
Comment (thaJeztah) [inline docs/reference/commandline/index.md]: We discussed this at some point, and we think that for the _index_, it's not needed to explain that one _or multiple_ objects can be passed to some commands, so those changes are not needed (on this line, and the other ones)

File: docs/reference/commandline/index.md [hunk 2]
```diff
@@ -134,9 +134,9 @@ read the [`dockerd`](dockerd.md) reference page.
 | Command | Description                                                        |
 |:--------|:-------------------------------------------------------------------|
 | [service create](service_create.md) | Create a new service                   |
-| [service inspect](service_inspect.md) | Inspect a service                    |
+| [service inspect](service_inspect.md) | Display information on one or more services |
 | [service ls](service_ls.md) | List services in the swarm                     |
-| [service rm](service_rm.md) | Reemove a swervice from the swarm              |
-| [service scale](service_scale.md) | Set the number of replicas for the desired state of the service |
+| [service rm](service_rm.md) | Remove one or more services                    |
```
Comment (thaJeztah) [inline docs/reference/commandline/index.md]: "Remove a swervice", haha that made me laugh :joy:

**Review by thaJeztah — CHANGES_REQUESTED** [review]
Sorry @yuexiao-wang, but as explained in my first comment, I think we don't need to change those descriptions.

We _do_ need to fix that typo though :smile:

**Comment by vdemeester** [comment]
Same comments as @thaJeztah 👼

**Comment by yuexiao-wang** [comment]
@thaJeztah  Got it. Should "Reemove a swervice " be changed?
Thanks!

**Comment by thaJeztah** [comment]
> Should "Reemove a swervice " be changed?

Yes please! :smile_cat:

**Comment by yuexiao-wang** [comment]
@thaJeztah @vdemeester PTAL
It is done to update this PR.
Thanks!

**Review by vdemeester — APPROVED** [review]
LGTM 🐸

**Comment by thaJeztah** [comment]
swexcellent swthank you! :wink:

LGTM

**Pull request merged**